### ISSUE TITLE
Language syntax does not support digit

### ISSUE THREAD

**Issue by chuzui** [dgraph-io/dgraph#1863]
Hi
For querying the object by alias, I build some rdf data as blow
```
<MS> <name> "microsoft"@en .
<MS> <name> "ms"@a1 . (a1 is for "alias 1")
```
Then the query such like 'eq(name@:, "ms")' works well.
But when i explicitly write the "name@a1", i got a error "Expected comma or language but got: 1.".
I checked the source code and found that the language syntax does not support digit in query. So the behavior of language syntax between loading data and query are inconsistent.

**Comment by peterstace** [comment]
Thank you for reporting, consistency between query and loading is important. I'll take a look into this.

**Comment by peterstace** [comment]
I was able to reproduce this pretty easily.

**Comment by peterstace** [comment]
Fixed in fc2c509
